framework/src/Volo.Abp.MemoryDb/Volo/Abp/Domain/Repositories/MemoryDb/MemoryDbRepository.cs

```diff
@@ -309,5 +310,18 @@ public virtual async Task DeleteAsync(TKey id, bool autoSave = false, Cancellati
         {
             await DeleteAsync(x => x.Id.Equals(id), autoSave, cancellationToken);
         }
+
+        public virtual async Task DeleteManyAsync([NotNull] IEnumerable<TKey> ids, bool autoSave = false, CancellationToken cancellationToken = default)
```

Comment:
This can be default implementation in the RepositoryBase, so we don't need to implement here.

Reply:
Can you check Commit f3722985